### PR TITLE
summary store / maps

### PR DESCRIPTION
**Pull request by kkraune**
work in progress but good for now

### CODE REVIEW

File: documentation/performance/sizing-feeding.html [hunk 1]
```diff
@@ -247,3 +247,52 @@ <h3 id="cluster-out-of-sync">Cluster out of sync</h3>
 vds.idealstate.merge_bucket.done_failed
 </pre>
 </p>
+
+
+<h3 id="summary-store">Summary store</h3>
```
Comment (geirst) [inline documentation/performance/sizing-feeding.html]: @kkraune We should call this section "Document Store".

File: documentation/performance/sizing-feeding.html [hunk 2]
```diff
@@ -247,3 +247,52 @@ <h3 id="cluster-out-of-sync">Cluster out of sync</h3>
...
+If the <a href="../document-summaries.html#document-store">document store</a>
+is accessed when evaluating the condition, performance drops.
+Conditions should be evaluated using attribute values for high performance -
+in the example above, <em>myField</em> should be an attribute.
+</p><p>
+Note: If the condition uses struct or map, values are read from the document store:
+<pre>
+    "condition": "myDoc.myMap{1} == 3"
+</pre>
+This is true even though all struct/map elements are defined as attribute.
```
Comment (geirst) [inline documentation/performance/sizing-feeding.html]: Here we should replace "struct/map elements" with "struct fields", as that is how it it specified and named in the sd file.

File: documentation/performance/sizing-feeding.html (hunk 1 above)
Comment (kkraune) [reply documentation/performance/sizing-feeding.html]: agreed! wil fix

File: documentation/performance/sizing-feeding.html (hunk 2 above)
Comment (kkraune) [reply documentation/performance/sizing-feeding.html]: agreed

**Pull request merged**